### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by cayop**
There are some in the INSTALLDEV.md for the implementation for develop works with dockers.
I Changed the Dockerfile because we need fixed the version of the operating system. Actually it is debian 10.
And we need install python3.6 in that version.

I modify orchestra-admin and orchestra-deploy for automatized the process for install it.
I'm not try run the tests. And maybe when we see the problems with the test we need return to modify the implementation of this process.

### CODE REVIEW

**Review by slamora — CHANGES_REQUESTED** [review]
Can you check if `scripts/containers/orchestra-deploy` is a copy of https://raw.githubusercontent.com/ribaguifi/django-orchestra/master/scripts/containers/deploy.sh ??

**Comment by cayop** [comment]
It's true, I moved it.
And I changed the rest of the commands of the orchestra of "orchestra/management/commands" dir

**Comment by cayop** [comment]
It's true, I moved it.
And I changed the rest of the commands of the orchestra of "orchestra/management/commands" dir

File: scripts/containers/deploy.sh [hunk 1]
```diff
@@ -46,13 +46,16 @@ function install_orchestra () {
     
     if [[ $dev ]]; then
         # Install from source
-        python_path=$(python3 -c "import sys; print([path for path in sys.path if path.startswith('/usr/local/lib/python')][0]);")
+        python_path=$(python3 -c "import sys; print([path for path in sys.path if path.startswith('/usr/local/lib/python')][1]);")
         if [[ -d $python_path/orchestra ]]; then
             run sudo rm -fr $python_path/orchestra
         fi
-        orch_version=$(python3 -c "from orchestra import get_version; print(get_version());" 2> /dev/null)
+	orch_version=$(python3 -c "from orchestra import get_version; print(get_version());" 2> /dev/null || echo '')
```
Comment (slamora) [inline scripts/containers/deploy.sh:53]: Please, check the indentation.

File: scripts/containers/deploy.sh [hunk 2]
```diff
@@ -46,13 +46,16 @@ function install_orchestra () {
...
         if [[ ! $orch_version ]]; then
             # First Orchestra installation
+	    run sudo mkdir -p /usr/share/man/man1
+	    run sudo mkdir -p /usr/share/man/man7
+	    run sudo apt-get update
```
Comment (slamora) [inline scripts/containers/deploy.sh:58]: Underindented lines?

File: scripts/containers/deploy.sh [hunk 3]
```diff
@@ -65,26 +68,26 @@ function install_orchestra () {
         if [[ -L /usr/local/bin/orchestra-admin || -f /usr/local/bin/orchestra-admin ]]; then
             run sudo rm -f /usr/local/bin/{orchestra-admin,orchestra-beat}
         fi
-        run sudo ln -s $home/django-orchestra/orchestra/bin/orchestra-admin /usr/local/bin/
-        run sudo ln -s $home/django-orchestra/orchestra/bin/orchestra-beat /usr/local/bin/
+	run sudo pip3 install -e $home/django-orchestra
```
Comment (slamora) [inline scripts/containers/deploy.sh:71]: Underindented lines?

File: scripts/containers/deploy.sh [hunk 4]
```diff
@@ -65,26 +68,26 @@ function install_orchestra () {
...
     noinput=$2
-    run sudo apt-get install -y postgresql python3-psycopg2
+    run sudo apt-get install -y postgresql
+    run sudo pip install psycopg2
     # Setup Database
     if [[ $dev ]]; then
         # Speeding up tests, don't do this in production!
         . /usr/share/postgresql-common/init.d-functions
-        pg_version=$(psql --version | head -n1 | sed -r "s/^.*\s([0-9]+\.[0-9]+).*/\1/")
+	pg_version=$(psql --version | head -n1 | awk '{print $3}' | sed -e "s,\..*,,")
```
Comment (slamora) [inline scripts/containers/deploy.sh:90]: Underindented lines?

**Review by slamora — CHANGES_REQUESTED** [review]

**Pull request merged**